CONTRIBUTING.md

```diff
@@ -10,6 +10,21 @@ using [Azure/autorest.go][] and [Azure/autorest][]. These generated packages dep
 
 For bugs or feature requests you can submit them using the [Github issues page][issues] and filling the appropriate template.
 
+## Building
+
+SDKs are either "track 1" or "track 2":
+
+- Track 1 SDKs are found in the services/ and profiles/ top level folders. 
+    - CI is in \azure-pipelines.yml
+- Track 2 SDK are foudn in the sdk/ top level folder.
+    - CI is in \eng\pipelines\templates\steps\build.yml
+
+To build, run `go build` from the respective SDK directory.
+
+There currently is not a repository wide way to build or regenerate code.
+
+## Guidelines and Code of Conduct
+
 Also please see these [guidelines][] about contributing to Azure projects.
```

Comment:
You didn't touch this, but maybe we could include the actual link about contributing to Azure projects